server/boot/user.js

```diff
@@ -223,6 +225,18 @@ module.exports = function(app) {
   );
 
   router.get('/:username', showUserProfile);
+  router.get(
+    '/:username/report-user/',
+    ifNoUser401,
```

Comment:
sending status code should be done when the endpoint is an api. 

When the endpoint is sending back to a page the redirect should also send them to a page.

So here we want to redirect unauthenticated(not logged in) users to either the `map` or `login`